src/components/Dashboard/Utilization/UtilizationItem.js

```diff
@@ -50,7 +50,7 @@ export class UtilizationItem extends React.PureComponent {
     if (isLoading) {
       chart = <LoadingComponent />;
     } else if (data) {
-      const chartData = data.map((val, index) => ({ x: index, y: Number(val.toFixed(1)) }));
+      const chartData = data.map((val, index) => ({ x: index, y: Number(val && val.toFixed(1)) }));
```

Comment:
how is this `x`  without `y` value going to visualize in graph ? I think it would be better to first filter all data without value

Reply:
I would move this logic upwards, i.e. to `getUtilizationVectorStats()` or to other place where the source data are being parsed.

Please note, numeric zero `0` is valid value here but it is `false` if transformed to boolean.

Reply:
`parseNumber` from `utils` can be used here

Reply:
IIUC, `val` can be null. In that case `null.toFixed(1)` will fail.
In other flows, the `y` can become `null` which is not preferred either.

**I still prefer moving this logic to the "parser".**
Or at least: `y: val ? Number(val.toFixed(1)) : 0` . This is no 100% but covers expected cases (expects `val` to be numeric or falsish).